test/externalTests/gnosis.sh

```diff
@@ -47,7 +47,10 @@ function gnosis_safe_test
         "${compile_only_presets[@]}"
         #ir-no-optimize            # Compilation fails with "YulException: Variable var_call_430_mpos is 1 slot(s) too deep inside the stack."
         #ir-optimize-evm-only      # Compilation fails with "YulException: Variable var_call_430_mpos is 1 slot(s) too deep inside the stack."
-        ir-optimize-evm+yul
+        # TODO: Uncomment the preset below when one of the following issues is solved:
+        # - https://github.com/safe-global/safe-contracts/issues/544
+        # - https://github.com/ethereum/solidity/issues/13972
```

Comment:
As I just commented, #13792 is not really a solution after all. Can you remove this line?

Reply:
By the way, you put the link to an unrelated PR with the same issue number. But I did the requested change ;)

Reply:
Haha, typo :) It was supposed to be #13972.